cvat/apps/organizations/serializers.py

```diff
@@ -62,7 +60,7 @@ class InvitationReadSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Invitation
-        fields = ['key', 'created_date', 'owner', 'role', 'user', 'organization']
+        fields = ['key', 'created_date', 'owner', 'role', 'user', 'organization', 'expired']
```

Comment:
I think its good for user to see that there was an invitation but its expired. UI can show expired invitations. Otherwise user can think that there was an error and invitaion was never created.